### ISSUE TITLE
Installing multiple extentions for code-server module doesn't work

### ISSUE THREAD

**Issue by Conor-Behard333** [coder/modules#89]
I am currently getting the following issue when trying to install multiple extension using the code-server module:
```
Installing code-server! 
################################################
🥳 code-server has been installed in /tmp/code-server 
🧩 Installing extension ms-azuretools.vscode-docker,codezombiech.gitignore... Extension 'ms-azuretools.vscode-docker,codezombiech.gitignore' not found. Make sure you use the full extension ID, including the publisher, e.g.: ms-dotnettools.csharp Failed Installing Extensions: ms-azuretools.vscode-docker,codezombiech.gitignore
```
my module terraform code is the following:
```
module "code-server" { 
    source = "https://registry.coder.com/modules/code-server" 
    agent_id = coder_agent.main.id 
    settings = { "workbench.colorTheme" = "Default Dark+" } 
    extensions = [ "ms-azuretools.vscode-docker", "codezombiech.gitignore" ]
 }
```

**Comment by stirby** [comment]
I was able to reproduce this. Working on a solution now.

**Comment by stirby** [comment]
Fixed in #90 with the help of @kylecarbs.
